include/deal.II/dofs/dof_renumbering.h

```diff
@@ -1273,6 +1273,36 @@ namespace DoFRenumbering
     std::vector<types::global_dof_index> &new_dof_indices,
     const DoFHandler<dim, spacedim>      &dof_handler);
 
+  /**
+   * Reorder the degrees of freedom in lexicographic order of support points.
+   *  The tolerance used to determine whether two support points are equal.
+   *
+   * This is particularly useful for finite element spaces that are defined on
+   * tensor product grids, such as the ones generated by
+   * GridGenerator::hyper_rectangle() or GridGenerator::hyper_cube() or
+   * their subdivided variants.
+   * With this ordering certain higher-dimension finite element matrices can be
+   * expressed as Kronecker products of lower-dimensional matrices.
+   *
+   * @warning This function is intended to be used on a small grid.
```

Comment:
I assume that's because the function is expensive? Perhaps that would be useful to say as well (including, if it's obvious to you, what the complexity in the number of unknowns N is).

Reply:
It is not expensive, but to support distributed grid one should implement some communication that is not the case. Technically using this function for larger meshes should be ok. I will update documentation

Reply:
I think it would be good if you add the aspect of parallelization aspect.